engine/engine_test.go

```diff
@@ -117,6 +116,16 @@ func TestQueriesAgainstOldEngine(t *testing.T) {
 		end   time.Time
 		step  time.Duration
 	}{
+		{
```

Comment:
technically those are in the fuzzing corpus, i just minimized them manually so i could fix them more easily